docs/05-audio-interfaces.md

```diff
@@ -60,8 +60,8 @@ These are the DACs that are known to work with balenaSound. If your DAC is not o
 | [Hifiberry DAC+ DSP][23]      | hifiberry-dac                         | Yes
 | [Hifiberry MiniAmp][24]       | hifiberry-dac                         | Yes
 | [Hifiberry DAC2 HD][27]       | hifiberry-dacplushd                   | Yes
-| [Adafruit I2S Audio Bonnet][26]| "hifiberry-dac”, “i2s-mmap”          | Yes
-| [Adafruit MAX98357 I2S Class-D Mono Amp][28]| "hifiberry-dac”, “i2s-mmap”          | Yes
+| [Adafruit I2S Audio Bonnet][26]| "hifiberry-dac","i2s-mmap"           | Yes
+| [Adafruit MAX98357 I2S Class-D Mono Amp][28]| "hifiberry-dac”,"i2s-mmap" | Yes
```

Comment:
These values were using chars [U+201C](https://www.fileformat.info/info/unicode/char/201c/index.htm) and [U+201D](https://www.fileformat.info/info/unicode/char/201d/index.htm) instead of ASCII character 34 (0x22) for the double quote. This can be a problem if users literally copy and paste the values from this table.

I have also deleted the blank space after the comma because the following documentation omits the blank space:  
https://www.balena.io/docs/reference/OS/advanced/#setting-device-tree-overlays-dtoverlay-and-parameters-dtparam 
... but I don't know that removing the space is strictly necessary.

Reply:
good catch @pdcastro !